src/test/starknet.t.sol

```diff
@@ -58,6 +60,7 @@ interface StarknetDaiBridgeLike {
     function dai() external returns (address);
     function starkNet() external returns (address);
     function escrow() external returns (address);
+    function l2DaiBridge() external returns (uint256);
```

Comment:
Same as above, return uint256 or bytes32?

Reply:
uint256, it is declared this way on the bridge: https://github.com/makerdao/starknet-dai-bridge/blob/main/contracts/l1/L1DAIBridge.sol#L96